### PR TITLE
Added the driver option to run functional tests

### PR DESCRIPTION
**Pull request by yitam**
Added the Driver option to the functional tests
Cleaned up pdo_sqlsrv\MsCommon.inc
Fixed the two test_ae_keys_setup tests

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/793)

### CODE REVIEW

File: test/functional/pdo_sqlsrv/MsCommon.inc [hunk 1]
```diff
@@ -8,30 +8,6 @@
```
Comment (david-puglielli) [inline test/functional/pdo_sqlsrv/MsCommon.inc]: Where is the code below going? Is it already in MsCommon_mid-refactor.inc?

File: test/functional/pdo_sqlsrv/test_ae_keys_setup.phpt [hunk 2]
```diff
@@ -1,36 +1,49 @@
...
+$conn = connect();
+    
+if (isAEQualified($conn)){
+    $query = "SELECT name FROM sys.column_master_keys";
+    $stmt = $conn->query($query);
+
+    // Do not assume the master key must be the first one created
+    $found = false;
+    while ($master_key_row = $stmt->fetch()) {
+        if ($master_key_row[0] == 'AEMasterKey') {
```
Comment (david-puglielli) [inline test/functional/pdo_sqlsrv/test_ae_keys_setup.phpt]: These key names are the defaults? Are these the names they get created with in our setup scripts?

File: test/functional/pdo_sqlsrv/MsCommon.inc (hunk 1 above)
Comment (yitam) [reply test/functional/pdo_sqlsrv/MsCommon.inc]: yes, these were old and not used anymore

File: test/functional/pdo_sqlsrv/test_ae_keys_setup.phpt (hunk 2 above)
Comment (yitam) [reply test/functional/pdo_sqlsrv/test_ae_keys_setup.phpt]: yes, by the AE setup scripts

**Pull request merged**